examples/import-in-node/src/index.ts

```diff
@@ -0,0 +1,119 @@
+import * as dotenv from "dotenv";
+import * as path from "path";
+import { input } from "@inquirer/prompts";
+import { Turnkey } from "@turnkey/sdk-server";
+import { Crypto } from "@peculiar/webcrypto";
+import {
+  decodeKey,
+  hpkeEncrypt,
+  verifyEnclaveSignature,
+} from "@turnkey/crypto";
+import { uint8ArrayFromHexString } from "@turnkey/encoding";
+global.crypto = new Crypto();
+
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+async function main() {
+  const organizationId = process.env.ORGANIZATION_ID!;
+  const userId = process.env.USER_ID!;
+  const turnkeyClient = new Turnkey({
+    apiBaseUrl: process.env.BASE_URL!,
+    apiPublicKey: process.env.API_PUBLIC_KEY!,
+    apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    defaultOrganizationId: organizationId,
+  });
+  const importType = await input({
+    message: `Enter Import Type, either "wallet" or "key"`,
+  });
+  let initResult;
+  if (importType == "wallet") {
+    initResult = await turnkeyClient.apiClient().initImportWallet({
+      userId,
+    });
+  } else if (importType == "key") {
+    initResult = await turnkeyClient.apiClient().initImportPrivateKey({
+      userId,
+    });
+  } else {
+    throw new Error(`Invalid import type. Please enter "wallet" or "key"`);
+  }
+  const importBundle = JSON.parse(initResult.importBundle);
+
+  const verified = await verifyEnclaveSignature(
+    importBundle.enclaveQuorumPublic,
+    importBundle.dataSignature,
+    importBundle.data,
+  );
+  if (!verified) {
+    throw new Error(`failed to verify enclave signature: ${importBundle}`);
+  }
+
+  const signedData = JSON.parse(
+    new TextDecoder().decode(uint8ArrayFromHexString(importBundle.data))
+  );
+
+  if (
+    !signedData.organizationId ||
+    signedData.organizationId !== organizationId
+  ) {
+    throw new Error(
+      `organization id does not match expected value. Expected: ${organizationId}. Found: ${signedData.organizationId}.`
+    );
+  }
+  if (!signedData.userId || signedData.userId !== userId) {
+    throw new Error(
+      `user id does not match expected value. Expected: ${userId}. Found: ${signedData.userId}.`
+    );
+  }
+
+  if (!signedData.targetPublic) {
+    throw new Error('missing "targetPublic" in bundle signed data');
+  }
+
+  // Load target public key generated from enclave and set in local storage
+  const targetKeyBuf = uint8ArrayFromHexString(signedData.targetPublic);
+  if (importType == "wallet") {
+    const mnemonic = await input({
+      message: "Enter mnemonic seed phrase for wallet to import",
+    });
+    const plainTextBuf = new TextEncoder().encode(mnemonic);
+    const walletBundle = hpkeEncrypt({ plainTextBuf, targetKeyBuf });
```

Comment:
This **whole** section is very very security sensitive so I don't think we should leave that up to users to copy/paste. IMO the interface should be reduced to:
```ts
// This parses the bundles, verifies the signature and organization ID, then calls hpkeEncrypt
const walletBundle = const crypto.ImportWalletWithBundle(plainTextBuf, initResult.importBundle, userId, organizationId)
```

And in the case of private keys:
```ts
// does that same for private keys, including taking care of key format shenanigans
const keyBundle = const crypto.ImportPrivateKeyWithBundle(plainTextBuf, format, initResult.importBundle, userId, organizationId)
```

Basically let's make this as foolproof as possible.